translations/messages.fr.xlf

```diff
@@ -306,6 +306,78 @@
         <source>Align your pieces to score points, but beware, pieces can flip!</source>
         <target>Alignez vos pièces pour marquer des points, mais attention aux retournements de pièces !</target>
       </trans-unit>
+      <trans-unit id="8720126581400014790" datatype="html">
+        <source>Value is mandatory</source>
```

Comment:
soit: value mandatory, soit: this value is mandatory (et accorder le français avec ce choix si c'est celui-ci qui est pris)